src/lib/components/chat/Navbar.svelte

```
@@ -50,38 +52,78 @@
+							{:else}
+								&#8226; {$i18n.t('This is a limited version of the App. To experience full capability')}<br>
+							{/if}
+						</span>
+						<button
+							class="bg-[#EB8486] hover:bg-[#EB5352] text-white/60 hover:text-white transition w-max rounded-full font-medium text-sm py-1 px-4"
+							on:click={async () => {
+								await userSignOut();
+								user.set(null);
+								localStorage.removeItem('token');
+								location.href = '/auth';
+								show = false;
```

ESLint

[error] 'show' is not defined.

(no-undef)